x-pack/plugins/spaces/index.js

```diff
@@ -42,10 +42,11 @@ export const spaces = (kibana) => new kibana.Plugin({
     hacks: [],
     mappings,
     home: ['plugins/spaces/register_feature'],
-    injectDefaultVars: function () {
+    injectDefaultVars: function (server) {
       return {
         spaces: [],
-        activeSpace: null
+        activeSpace: null,
+        spaceSelectorURL: server.config().get('server.basePath') || '/',
```

Comment:
Why inject this? I think it would make more sense to use `chrome.addBasePath('/')` clientside

Reply:
`chrome.addBasePath()` will include the current space identifier as part of the URL, so if we are in the `marketing` space, then `chrome.addBasePath('/')` will result in `/serverBasePath/s/marketing`, instead of just `/serverBasePath`